guidelines/index.html

```diff
@@ -2975,10 +2975,10 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
    
 </dd>
             
-            <dt class="new"><dfn id="dfn-perimeter" tabindex="0" aria-haspopup="dialog" data-dfn-type="dfn">perimeter</dfn></dt>
+            <dt class="new"><dfn id="dfn-perimeter" tabindex="0" aria-haspopup="dialog" data-dfn-type="dfn">外周 (perimeter)</dfn></dt>
```

Comment:
Wikipediaいわく、数学的には[周長](https://ja.wikipedia.org/wiki/%E5%91%A8%E9%95%B7)になりますかね…？
```suggestion
            <dt class="new"><dfn id="dfn-perimeter" tabindex="0" aria-haspopup="dialog" data-dfn-type="dfn">周長 (perimeter)</dfn></dt>
```

Reply:
ここを「周長」にすると、リンク元のテキスト（具体的には一箇所のみで、達成基準 2.4.13 Focus Appearance の要件の箇条書きのひとつめ）を以下のように修正する必要があります。

> フォーカスされていないコンポーネント又はサブコンポーネントの、太さ 2 CSS ピクセルの**外周**の領域と同等以上の大きさである。

↓

> フォーカスされていないコンポーネント又はサブコンポーネントの、太さ 2 CSS ピクセルの**周長**の領域と同等以上の大きさである。

Reply:
@bakera 
どうしましょう…？

Reply:
Wikipediaいわく

>英語の perimeter は周囲と周長の両方を指す。

ということなのでややこしいですね。リンク元の文は「周囲」でないと意味が通じないものかと思います。この文の perimeter も周囲で良いかと。その一方、次のexampleにある"perimeter calculation"は2つの長さを比較する際に必要になる長さの計算について述べていますから、こちらは周長でないと意味が通じないですね。
原文は同じ言葉ですが訳し分けるしかないかなぁと。